src/features/debugger/debugMessage.ts

```diff
@@ -240,8 +240,18 @@ async function checkMultisigIsmEmpty(
     throw new Error('Recipient ISM is not a valid address');
   }
   const ism = InterchainSecurityModuleFactory.connect(ismAddress, destProvider);
-  const moduleType = await ism.moduleType();
 
+  let moduleType: number | undefined = undefined;
+  try {
+    moduleType = await ism.moduleType();
```

Comment:
Another way of checking this would be to do `destProvider.getCode(ism)` and see that it is not `0x0` (unless `ism == 0x0` itself)